style.css

```diff
@@ -0,0 +1,465 @@
+
+#location p {
+  font-size: 14px;
+  font-weight: 700;
+  color: #868686;
+}
+
+/* Program Section */
+#program {
+  padding-top: 31px;
+  padding-bottom: 31px;
+  background-color: #282b32;
+  color: #fff;
+  background-image: url(asset/diamond.png);
+}
+
+.dash {
+  background-color: #a04038;
+  width: 30px;
+  height: 3px;
+  border: none;
+  margin: 15px 0 40px 0;
+}
+
+.program-items {
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+}
+
+.program-items > div {
+  display: grid;
+  grid-template-columns: 20% 20% 60%;
+  background-color: rgba(255, 255, 255, 0.1);
+  padding: 15px 9px;
+  align-items: center;
+}
+
+h6 {
+  color: #ff5438;
+  font-size: 15px;
+}
+
+.btn {
+  padding: 20px 50px;
+  background-color: #ff4f38;
+  margin-top: 40px;
+  border: none;
+  cursor: pointer;
+  color: #fff;
+  font-size: 16px;
+  font-weight: 900;
+}
+
+#program a {
+  display: none;
+}
+
+/* Featured Speakers */
+#speaker-section {
+  padding-top: 15px;
+  padding-bottom: 33px;
+}
+
+#speaker-card-container {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: repeat(6, 1fr);
+  row-gap: 18px;
+}
+
+.speaker-info {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 100px auto;
+  grid-template-rows: auto;
+  column-gap: 15px;
+}
+
+.speaker-info span {
+  color: #f78c89;
+}
+
+.speaker-info .dash {
+  background-color: #d9d9d9;
+  margin: 6px 0 13px 0;
+}
+
+.speaker-img-container {
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+}
+
+.speaker-img-container img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+#desktop-sponsors {
+  display: none;
+}
+
+/* Footer Section */
+#footer-desktop {
+  display: none;
+}
+
+.footer-container > div:nth-child(1) {
+  width: 30%;
+}
+
+/* Media Queries */
+@media screen and (min-width: 768px) {
+  .mobile-main-menu {
+    display: none;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .overlay-show {
+    display: none;
+  }
+
+  /* Header Section */
+  header {
+    display: block;
+    width: 100%;
+    position: fixed;
+  }
+
+  #top-bar {
+    width: 100%;
+    background-color: #3e3e3e;
+    padding: 0.8% 14% 0.8% 0;
+  }
+
+  #top-bar ul {
+    display: flex;
+    justify-content: end;
+    gap: 20px;
+  }
+
+  #program a {
+    display: block;
+    color: #fff;
+    text-transform: uppercase;
+    margin-top: 60px;
+  }
+
+  #top-bar ul li a {
+    text-decoration: none;
+    color: #fff;
+  }
+
+  header .logo-img-container img,
+  footer .logo-img-container img {
+    width: 40px;
+  }
+
+  header .logo-img-container h2,
+  footer .logo-img-container h2 {
+    font-size: 1rem;
+  }
+
+  #navbar {
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    background-color: #fff;
+  }
+
+  .footer-container .logo-img-container {
+    width: 30%;
+  }
+
+  #navbar .logo-img-container {
+    text-decoration: none;
+    color: #000;
+  }
+
+  #navbar nav ul {
+    display: flex;
+    gap: 40px;
+  }
```

Comment:
- [ ] Well done, you did your best and made the changes asked by your previous code reviewer but as I can clearly see while making the changes somewhere you messed up with your design or `stylesheet`. Your desktop navigation looks unprofessional and bad to the eyes. Kindly fix it, give exactly the same style as you gave to the `navigation` menu for the `about.html` page. 

### Problem

![Screenshot from 2023-05-24 09-16-35](https://github.com/over-geek/capstone-project-2/assets/66476329/8ed7108d-98cd-4f81-b818-7123ad2e5943)